docs/book/features/vsphere_csi_vmc.md

```diff
@@ -0,0 +1,85 @@
+<!-- markdownlint-disable MD033 -->
+<!-- markdownlint-disable MD034 -->
+# vSphere CSI Driver - VMware Cloud on AWS (VMC) support
+
+## Introduction <a id="introduction"></a>
```

Comment:
I think you missed to add links on the top of the page which uses these anchors - `<a id="introduction"></a>`

Reply:
Fixed.